spec/factories/users.rb

```diff
@@ -177,6 +177,10 @@
 
     trait :signed_up do
```

Comment:
what if we renamed this for clarity/consistency?
```suggestion
    trait :fully_registered do
```

Reply:
> what if we renamed this for clarity/consistency?

Yeah, that seems like a good idea, though I suspect it might touch a _ton_ of files. I'd probably create a quick follow-on pull request for that.

Reply:
Follow-up at #8302